package.json

```diff
@@ -56,6 +56,52 @@
     "onStartupFinished"
   ],
   "main": "./out/extension.js",
+  "runme": {
+    "features": {
+      "ForceLogin": {
+        "enabled": true,
+        "conditions": {
+          "os": "All",
+          "vsCodeVersion": ">=1.58.0",
+          "enabledForExtensions": [
+            "stateful.platform"
```

Comment:
@sourishkrout I see your point, I agree. I've been thinking more in the simple way to match the extension IDs, but it will be more difficult for overrides purposes. Likewise, I'll follow your suggestion.

Reply:
I think there's some beauty in explicitly expressing true and false versus using the absence. It's a matter of preference, I suppose.